cub/benchmarks/bench/scan/exclusive/base.cuh

```diff
@@ -131,13 +131,7 @@ static void basic(nvbench::state& state, nvbench::type_list<T, OffsetT>)
   });
 }
 
-#ifdef TUNE_OffsetT
-using some_offset_types = nvbench::type_list<TUNE_OffsetT>;
-#else
-using some_offset_types = nvbench::type_list<uint32_t, uint64_t>;
-#endif
-
-NVBENCH_BENCH_TYPES(basic, NVBENCH_TYPE_AXES(all_types, some_offset_types))
+NVBENCH_BENCH_TYPES(basic, NVBENCH_TYPE_AXES(all_types, offset_types))
```

Comment:
Just to make sure I'm not missing something: `DeviceScan` is using `choose_offset_t` to map the user-provided offset type to an unsigned offset typ that is then used to instantiate the `DispatchScan`. Wouldn't we want to benchmark and tune for _unsigned_ offset types in that case? My intuition would have been to map the signed offset types from the axis to unsigned ones using `choose_offset_t` in the benchmark implementation for `scan`:
```
  using offset_t    = detail::choose_offset_t<OffsetT>;
```

Reply:
That's odd then. The documentation of `DispatchScan` says that `OffsetT` is a "Signed integer type for global offsets".

Reply:
Added your suggestion:

>  using offset_t    = detail::choose_offset_t<OffsetT>;